L1Trigger/Phase2L1ParticleFlow/interface/L1NNTauProducer.hh

```diff
@@ -0,0 +1,39 @@
+#ifndef L1TRIGGER_PHASE2L1PARTICLEFLOW_L1NNTAU_H
```

Comment:
Include guard should have full file name i.e. L1NNTauProducer, not just L1NNTau

Also, this file suffix should be `.h`, not `.hh`.

Reply:
done